docs/styling/common-props.md

```diff
@@ -0,0 +1,241 @@
+        "values": ["wrap", "nowrap", "balance", "pretty"],
+        "link": "https://developer.mozilla.org/en-US/docs/Web/CSS/text-wrap",
+    },
+
+    "top / bottom / right / left": {
+        "description": "Sets the vertical / horizontal position of a positioned element. It does not effect non-positioned elements.",
+        "values": ["0", "4em", "10%", "20px"],
+        "link": "https://developer.mozilla.org/en-US/docs/Web/CSS/top",
+    },
+    "width": {
+        "description": "Sets an element's width",
+        "values": ["150px", "20em", "75%", "auto"],
+        "link": "https://developer.mozilla.org/en-US/docs/Web/CSS/width",
+    },
+    "white_space": {
+        "description": "Sets how white space inside an element is handled",
+        "values": ["normal", "nowrap", "pre", "break-spaces"],
+        "link": "https://developer.mozilla.org/en-US/docs/Web/CSS/white-space",
+    },
+    "word_break": {
+        "description": "Sets whether line breaks appear wherever the text would otherwise overflow its content box",
+        "values": ["normal", "break-all", "keep-all", "break-word"],
+        "link": "https://developer.mozilla.org/en-US/docs/Web/CSS/word-break",
+    },
+    "z_index": {
+        "description": "Sets the z-order of a positioned element and its descendants or flex and grid items, and overlapping elements with a larger z-index cover those with a smaller one",
+        "values": ["auto", "1", "5", "200"],
+        "link": "https://developer.mozilla.org/en-US/docs/Web/CSS/z-index",
+    },
+    
+
+}
+
+
+def show_props(key, props_dict):
+    prop_details = props_dict[key]
+    return rx.table.row(
+        rx.table.cell(
+            rx.link(
+                rx.hstack(
+                    rx.code(key),
+                    rx.icon("square_arrow_out_up_right", color=rx.color("mauve", 11), size=13, flex_shrink="0"),
+                    align="center"
+                ),
+                href=prop_details["link"],
+                is_external=True,
+            ), 
+            justify="start",),
+        rx.table.cell(prop_details["description"], justify="start",),
+        rx.table.cell(rx.hstack(*[rx.code(value) for value in prop_details["values"]], flex_wrap="wrap"), justify="start",),
+        # rx.table.cell(
+        #     rx.center(
+        #         rx.link(
+        #             rx.icon("square_arrow_out_up_right", color=rx.color("mauve", 11), size=18,), href=prop_details["link"], is_external=True,
+        #         ),
+        #         width="100%",
+        #     ),
+        # ),
+        justify="center",
+        align="center",
+        
+    )
+
+```
+
+
+Any [CSS](https://developer.mozilla.org/en-US/docs/Web/CSS) prop can be used in a component in Reflex. This is a short list of the most commonly used props. To see all CSS props that can be used check out this [documentation](https://developer.mozilla.org/en-US/docs/Web/CSS).
+
```

Comment:
Should add a line to the effect of `hyphens in CSS property names may be replaced by underscores to use as valid python identifiers`

Reply:
added